docs/api/web-contents.md

```diff
@@ -588,6 +588,15 @@ Emitted when media starts playing.
 
 Emitted when media is paused or done playing.
 
+#### Event: 'audio-state-changed'
+
+Returns:
+
+* `event` Event
+* `audible` boolean - Whether audio is currently playing.
+
+Emitted when media becomes audible or inaudible.
```

Comment:
Again a little imprecise for the same reason, but ... revising this in a way that still scans well is not obvious to me. For example this would be more correct but feels awkward:

```suggestion
Emitted when the first frame or child content starts playing, or when the last stops playing.
```

If you have a third idea then let's consider that; otherwise, maybe OK as-is

Reply:
I updated the property description but i think this is probably more understandable as-is 🤔